examples/MongoDB Atlas-GCP VPC Peering/cluster.tf

```diff
@@ -0,0 +1,40 @@
+
+resource "mongodbatlas_cluster" "cluster-test-single" {
```

Comment:
May want to provide a comment around what this cluster example shows too.  I like how complex it is but a new user could likely use a general description.

Reply:
add a single line comment for what this cluster is.